jaeger-core/README.md

```diff
@@ -0,0 +1,11 @@
+# Core #
+If you only want to do custom instrumentation using the core tracing functionality then add the
+following:
+```xml
+<dependency>
+    <groupId>com.uber.jaeger</groupId>
+    <artifactId>jaeger-core</artifactId>
+    <version>$jaegerVersion</version>
+</dependency>
+```
+
```

Comment:
Added some instructions. You are right in that it is the core of the library. I feel that the add on modules are more important because more people will be using the add-ons compared to core functionality directly. What is the advantage of having it on the main readme?